tests/scripts/all.sh

```diff
@@ -204,6 +204,108 @@ pre_initialize_variables () {
     done
 }
 
+# add_to_group GROUP_NAME COMPONENT_NAME
+# Naming conventions:
+# * GROUP_NAME=COMPONENT_NAME to run the component on its own.
+# * GROUP_NAME=_PREFIX to group all components with the same prefix.
+# * GROUP_NAME=_PREFIX_MISC to group some components with the same prefix.
+add_to_group ()
+{
+    if [[ -v groups["$1"] ]]; then
```

Comment:
Annoyingly, on OS X, the standard bash (3.2.57) doesn't support -v

Reply:
We don't really need this function on anything but Linux, but it's an easy fix, so done.